frontend/app/src/composables/accounts/blockie.ts

```diff
@@ -0,0 +1,175 @@
+import { type Ref } from 'vue';
+
+// Source: https://github.com/ethereum/blockies
+// The random number is a js implementation of the Xorshift PRNG
+export interface BlockieOptions {
+  /** seed used to generate icon data, default: random */
+  seed: string;
+  /** to manually specify the icon color, default: random */
+  color: string;
+  /** choose a different background color, default: random */
+  bgcolor: string;
+  /** width/height of the icon in blocks, default: 8 */
+  size: number;
+  /** width/height of each block in pixels, default: 4 */
+  scale: number;
+  /**
+   * each pixel has a 13% chance of being of a third color, default: random.
+   *
+   * Set to -1 to disable it.
+   *
+   * These "spots" create structures that look like eyes, mouths and noses.
+   */
+  spotColor: string;
+}
+
+const randSeed = new Array(4); // Xorshift: [x, y, z, w] 32 bit values
+
+const seedRand = (seed: string) => {
+  for (let i = 0; i < randSeed.length; i++) {
+    randSeed[i] = 0;
+  }
+  for (let i = 0; i < seed.length; i++) {
+    randSeed[i % 4] =
+      (randSeed[i % 4] << 5) - randSeed[i % 4] + seed.charCodeAt(i);
+  }
+};
+
+function rand() {
+  // based on Java's String.hashCode(), expanded to 4 32bit values
+  const t = randSeed[0] ^ (randSeed[0] << 11);
+
+  randSeed[0] = randSeed[1];
+  randSeed[1] = randSeed[2];
+  randSeed[2] = randSeed[3];
+  randSeed[3] = randSeed[3] ^ (randSeed[3] >> 19) ^ t ^ (t >> 8);
+
+  return (randSeed[3] >>> 0) / ((1 << 31) >>> 0);
+}
+
+export const createColor = () => {
```

Comment:
Also, this export if we are encapsulating the functionality, there is no need for `createColor` to be called outside of this composable.